include/pokedex.h

```diff
@@ -1,7 +1,7 @@
 #ifndef GUARD_POKEDEX_H
 #define GUARD_POKEDEX_H
 
-extern u8 gUnknown_030060B0;
+extern u8 gUnusedU8;
```

Comment:
This name should be more descriptive, as `gUnusedU8` could be literally anywhere